build.rs

```diff
@@ -1,3 +0,0 @@
-fn main() {
-    uniffi::generate_scaffolding("src/phoenix_channels_client.udl").unwrap();
-}
```

Comment:
Why can this be dropped completely instead of being conditional on the feature flag?

Reply:
This can be dropped because src/lib uses `uniffi::setup_scaffolding!("phoenix_channels_client");` rather than `uniffi::include_scaffolding!("phoenix_channels_client");` (this uses the name from the udl). Come to think of it, I should remove the UDL entirely.